ERCS/erc-x.md

```diff
@@ -0,0 +1,354 @@
+      type: uint256
+    - name: owner
+      type: address
+
+  outputs:
+    - name: isPending
+      type: bool
+```
+
+#### claimableCancelDepositRequest
+
+The amount of `assets` in Claimable cancelation state for the `owner` to claim.
+
+MUST NOT show any variations depending on the caller.
+
+MUST NOT revert unless due to integer overflow caused by an unreasonably large input.
+
+```yaml
+- name: pendingCancelDepositRequest
+  type: function
+  stateMutability: view
+
+  inputs:
+    - name: requestId
+      type: uint256
+    - name: owner
+      type: address
+
+  outputs:
+    - name: assets
+      type: uint256
+```
+
+#### claimCancelDepositRequest
+
+Claims the deposit cancelation Request with `requestId` and `owner`.
+
+Transfers `assets` to `receiver`.
+
+`owner` MUST equal `msg.sender` unless the `owner` has approved the `msg.sender` by some mechanism. 
+
+MUST emit the `ClaimCancelDepositRequest` event.
+
+```yaml
+- name: claimCancelDepositRequest
+  type: function
+  stateMutability: nonpayable
+
+  inputs:
+    - name: requestId
+      type: uint256
+    - name: receiver
+      type: address
+    - name: owner
+      type: address
+  outputs:
+```
+
+#### cancelRedeemRequest
+
+Submits a Request for asynchronous redeem cancelation. This places the Request in Pending state, with a corresponding increase in `pendingCancelRedeemRequest` for the full amount of the pending redeem Request. 
+
+When the cancelation is Pending, new redeem Requests are blocked and `requestRedeem` MUST revert.
+
+When the cancelation is Claimable, `claimableCancelRedeemRequest` will be increased for the `owner`. `claimCancelRedeemRequest` can subsequently be called by `owner` to receive `shares`. A Request MAY transition straight to Claimable state but MUST NOT skip the Claimable state.
+
+`owner` MUST equal `msg.sender` unless the `owner` has approved the `msg.sender` by some mechanism. 
+
+MUST emit the `CancelRedeemRequest` event.
+
+```yaml
+- name: cancelRedeemRequest
+  type: function
+  stateMutability: nonpayable
+
+  inputs:
+    - name: requestId
+      type: uint256
+    - name: owner
+      type: address
+  outputs:
+```
+
+#### pendingCanceRedeemRequest
+
+Whether the given `requestId` and `owner` have a pending redeem cancelation Request.
+
+MUST NOT show any variations depending on the caller.
+
+MUST NOT revert unless due to integer overflow caused by an unreasonably large input.
+
+```yaml
+- name: pendingCanceRedeemRequest
+  type: function
+  stateMutability: view
+
+  inputs:
+    - name: requestId
+      type: uint256
+    - name: owner
+      type: address
+
+  outputs:
+    - name: isPending
+      type: bool
+```
+
+#### claimableCancelRedeemRequest
+
+The amount of `shares` in Claimable cancelation state for the `owner` to claim.
+
+MUST NOT show any variations depending on the caller.
+
+MUST NOT revert unless due to integer overflow caused by an unreasonably large input.
+
+```yaml
+- name: pendingCancelRedeemRequest
```

Comment:
believe this should be claimableCancelRedeemRequest?

Reply:
Good catch! Fixed